app/components/create_card/index.js

```diff
@@ -67,6 +68,24 @@ class CreateCard extends Component {
 
     renderItem = ({item, index}) => {
         return (
+            index > 2 ?
+            <View style={styles.formItemContainer}>
+                <Text style={styles.formTitle}> {item.title} </Text>
+                <View style={styles.addAttributeContainer}>
+                    <TextInput
+                        style={styles.formInput}
+                        placeholder={"Enter " + item.title}
+                        underlineColorAndroid="transparent"
+                        onChangeText={(text) => this.handleFormTextChange(index, text)}
+                    />
+                    <View style={styles.addFieldButton}>
+                        <TouchableOpacity onPress={() => this.removeAttributeFromForm(item)}>
+                            <Text style={styles.buttonText}>Remove Attribute</Text>
```

Comment:
Consider renaming to just "Remove" so there is more room for users to type on the line